prow/oss/config.yaml

```diff
@@ -147,7 +147,7 @@ managed_webhooks:
       token_created_after: 2021-04-21T00:10:00Z
     kubeflow:
       token_created_after: 2020-12-06T00:10:00Z
-    looker:
+    looker/helltool:
```

Comment:
Do we need to disable webhooks and `in_repo_config` for the rest of the org? That will disable automation for all other repos. I think we can keep this as is and just tweak the `in_repo_config.allowed_clusters` to limit running jobs in the private build cluster to private repos and `plugins` to limit verbose/intrusive automation to the repos we want to enable it on.

Reply:
My impression is that the other repos might not be ready for prow yet, so might not worth spamming `hook` with those unused webhook events. WDYT?

Reply:
I wouldn't be worried about Hook. It can easily handle the webhook load and the plugins we configured shouldn't do anything unless they are needed with the exception of `owners-label` and `verify-owners`. Those will shallow clone the repos before seeing that there are no OWNERS files so those are the only plugins that we might want to disable org wide.

We can switch this back to org easily so this isn't a big deal. I was just thinking that it would be slightly preferable to continue supporting `/assign` and `/meow` on other repos.
/lgtm
/hold

Reply:
I actually agree with you. Updated to receive hook from the org